File: src/main/java/br/com/zup/academy/dominio/repository/ClienteRepository.java
package br.com.zup.academy.dominio.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import br.com.zup.academy.dominio.modelo.Cliente;

@Repository
public interface ClienteRepository extends JpaRepository<Cliente, Long>{

}

File: casa-do-codigo-grpc/src/main/kotlin/br/com/zup/dominio/repository/ClienteRepository.kt
package br.com.zup.dominio.repository

import br.com.zup.dominio.modelo.Cliente
import io.micronaut.data.annotation.Repository
import io.micronaut.data.jpa.repository.JpaRepository

@Repository
interface ClienteRepository: JpaRepository<Cliente, Long> {

}
